script/server

```diff
@@ -61,6 +61,18 @@ then
   fatal "config/diaspora.yml is missing! Copy over config/diaspora.yml.example to config/diaspora.yml and edit it properly!"
 fi
 
+# Check if detached head state
+if git status | grep 'Changes not staged for commit' > /dev/null ;
+then
+  warning "Your changes are not staged for commit. Kindly stash or commit them"
+fi
+
+# Check if bundle is complete
+if !( bundle check ) ;
```

Comment:
No need for the parens or the semicolon. you also should suppress its output IMO.